runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -205,20 +212,19 @@ private static SlotLimitationRequirement emptySlot(String description, Equipment
 		return new SlotLimitationRequirement(description, slots);
 	}
 
-	private final String text;
-	private final Integer stashUnit;
-	private final WorldPoint location;
-	private final Emote firstEmote;
-	private final Emote secondEmote;
-	@Nonnull
-	private final ItemRequirement[] itemRequirements;
+	private String text;
+	private STASHUnit stashUnit;
+	private WorldPoint location;
+	private Emote firstEmote;
+	private Emote secondEmote;
+	private ItemRequirement[] itemRequirements;
 
-	private EmoteClue(String text, Integer stashUnit, WorldPoint location, Emote firstEmote, @Nonnull ItemRequirement... itemRequirements)
+	private EmoteClue(String text, STASHUnit stashUnit, WorldPoint location, Emote firstEmote, ItemRequirement... itemRequirements)
```

Comment:
any reason why you removed nunnull annotation?

Reply:
Reverted, it was lost while merging manually